lib/MusicBrainz/Server/Controller/Search.pm

```diff
@@ -178,6 +178,9 @@ sub direct : Private
             $c->model('Event')->load_related_info(@entities);
             $c->model('Event')->load_areas(@entities);
         }
+        when ('tag') {
+            $c->model('Genre')->load(@entities);
+        }
```

Comment:
Won’t it fail on non primary tags?

Reply:
You mean on tags that are not genres? If so, it seemed to work fine when I tested it.

Reply:
On tags that are associated with genre but not as primary tag?

Reply:
Oh. We don't actually have those yet, so not sure :D

Reply:
Added a comment to remind us to support aliases once we add them